files/zh-cn/web/accessibility/aria/attributes/index.md

```diff
@@ -0,0 +1,130 @@
+    - [`aria-flowto`](/zh-CN/docs/Web/Accessibility/ARIA/Attributes/aria-flowto)
+    - [`aria-labelledby`](/zh-CN/docs/Web/Accessibility/ARIA/Attributes/aria-labelledby)
+    - [`aria-owns`](/zh-CN/docs/Web/Accessibility/ARIA/Attributes/aria-owns)
+    - [`aria-posinset`](/zh-CN/docs/Web/Accessibility/ARIA/Attributes/aria-posinset)
+    - [`aria-rowcount`](/zh-CN/docs/Web/Accessibility/ARIA/Attributes/aria-rowcount)
+    - [`aria-rowindex`](/zh-CN/docs/Web/Accessibility/ARIA/Attributes/aria-rowindex)
+    - [`aria-rowspan`](/zh-CN/docs/Web/Accessibility/ARIA/Attributes/aria-rowspan)
+    - [`aria-setsize`](/zh-CN/docs/Web/Accessibility/ARIA/Attributes/aria-setsize)
+
+## 全局 ARIA 特性
+
+无论是否应用了 ARIA 角色，某些状态和属性始终适用于所有的 HTML 元素。它们被定义为“全局”特性。所有角色和基本标记元素都支持全局状态和属性。
+
+上面所列出的许多特性都是全局的，这意味着除非被“特别禁止”，它们可以被包含在任意的元素中：
+
+- [`aria-atomic`](/zh-CN/docs/Web/Accessibility/ARIA/Attributes/aria-atomic)
+- [`aria-busy`](/zh-CN/docs/Web/Accessibility/ARIA/Attributes/aria-busy)
+- [`aria-controls`](/zh-CN/docs/Web/Accessibility/ARIA/Attributes/aria-controls)
+- [`aria-current`](/zh-CN/docs/Web/Accessibility/ARIA/Attributes/aria-current)
+- [`aria-describedby`](/zh-CN/docs/Web/Accessibility/ARIA/Attributes/aria-describedby)
+- [`aria-description`](/zh-CN/docs/Web/Accessibility/ARIA/Attributes/aria-description)
+- [`aria-details`](/zh-CN/docs/Web/Accessibility/ARIA/Attributes/aria-details)
+- [`aria-disabled`](/zh-CN/docs/Web/Accessibility/ARIA/Attributes/aria-disabled)
+- [`aria-dropeffect`](/zh-CN/docs/Web/Accessibility/ARIA/Attributes/aria-dropeffect)
+- [`aria-errormessage`](/zh-CN/docs/Web/Accessibility/ARIA/Attributes/aria-errormessage)
+- [`aria-flowto`](/zh-CN/docs/Web/Accessibility/ARIA/Attributes/aria-flowto)
+- [`aria-grabbed`](/zh-CN/docs/Web/Accessibility/ARIA/Attributes/aria-grabbed)
+- [`aria-haspopup`](/zh-CN/docs/Web/Accessibility/ARIA/Attributes/aria-haspopup)
+- [`aria-hidden`](/zh-CN/docs/Web/Accessibility/ARIA/Attributes/aria-hidden)
+- [`aria-invalid`](/zh-CN/docs/Web/Accessibility/ARIA/Attributes/aria-invalid)
+- [`aria-keyshortcuts`](/zh-CN/docs/Web/Accessibility/ARIA/Attributes/aria-keyshortcuts)
+- [`aria-label`](/zh-CN/docs/Web/Accessibility/ARIA/Attributes/aria-label)
+- [`aria-labelledby`](/zh-CN/docs/Web/Accessibility/ARIA/Attributes/aria-labelledby)
+- [`aria-live`](/zh-CN/docs/Web/Accessibility/ARIA/Attributes/aria-live)
+- [`aria-owns`](/zh-CN/docs/Web/Accessibility/ARIA/Attributes/aria-owns)
+- [`aria-relevant`](/zh-CN/docs/Web/Accessibility/ARIA/Attributes/aria-relevant)
+- [`aria-roledescription`](/zh-CN/docs/Web/Accessibility/ARIA/Attributes/aria-roledescription)
+
+除了 `aria-label` 和 `aria-labelledby` 这两个被“特别禁止”的属性，其它属性是全局适用的。这两个被“特别禁止”的属性无法在具有 [`presentation`](/zh-CN/docs/Web/Accessibility/ARIA/Roles/presentation_role) 角色或与其同义的 [`none`](/zh-CN/docs/Web/Accessibility/ARIA/Roles/none_role) 角色的元素上使用。
+
+## MDN 上介绍的状态和属性
+
+以下是 <abbr title="Mozilla Developer Network">MDN</abbr> 上介绍的 WAI-ARIA 状态和属性。
+
+{{SubpagesWithSummaries}}
```

Comment:
waiting for mving document from `Web/Accessibility/ARIA/ARIA_Techniques/Using_the_aria*`